### DISCUSSION TITLE
Private thread members

### DISCUSSION THREAD

**Discussion by overload08** [Rapptz/discord.py#9479] [Q&A]
Hi there !

Is ti possible to get all members objects of a private thread ?
When I use ctx.channel.members, it always return empty list

I have Intents.all() setted...

obj.member_count = 3
obj.members = []

**Comment by LostLuma** [comment]
As documented for [`Thread.members`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Thread.members), if the list of members is empty and/or incomplete you should call [`Thread.fetch_members`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Thread.fetch_members) instead.

**Reply by overload08** [reply]
It's working ! Thank's !
